data-raw/projects/proposals/ADDIA.yaml

```diff
@@ -0,0 +1,32 @@
+submitted_on: '2025-03-25'
+full_name: Signe Kaas Ovesen
+github_username: sioves
+job_position: PhD student
+phd_supervisor: Søren Gregersen, Daniel Ibsen, Erik Grove, Cecilia Bergh, Rikard Landberg
```

Comment:
Hmm, who is primary? Not sure how we want to deal with this in the future. List all of them on the website? Or...? Include "primary supervisor" and "others"?

Reply:
Søren is the main PhD supervisor.  The question could also just be "main phd supervisor." Then people know it is just one person. On the other hand, knowing all who is involved is also useful. So for me the entire supervisor team would be good to add - at least those you expect to be on the paper (=involved in this part of the project).

I also do not have any objections to the protocol and clearly see that it fits within our projects overall aim.